languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -35450,6 +35450,11 @@ USA
             <example correction="materials|matériels">I needed more electronic <marker>materiels</marker> to build a fully-functional machine.</example>
         </rule>
         <rule id="BARCA_DIACRITIC" name="Barca or Barça">
+            <antipattern>
+                <token skip="-1" postag="SENT_START"/>
+                <token skip="-1" regexp="yes">surname|cognomen|Hannibal|Hamilcar|Hasdrubal</token>
+                <token postag="SENT_END"/>
+            </antipattern>
             <pattern>
                 <token>
```

Comment:
For some reason, there are three lines that are not being shown here. The rule editor shows no more FPs if this part becomes:

```
<token>barca
                    <exception scope="previous" regexp="yes">da|la|cognomen|Hannibal|Hamilcar|Hasdrubal|Latin</exception>
                </token>
            </pattern>
```
Notice the `scope="previous"` and the added `Latin`.